prober/tcp.go

```diff
@@ -94,6 +94,10 @@ func ProbeTCP(ctx context.Context, target string, module config.Module, registry
 		Name: "probe_ssl_earliest_cert_expiry",
 		Help: "Returns earliest SSL cert expiry date",
 	})
+	probeSSLLatestVerifiedChainExpiry := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_ssl_latest_verified_chain_expiry",
```

Comment:
That's a little of a mouthful, and naming conventions have matured since the previous one was named.

I'd go for something like probe_ssl_last_chain_expiry_timestamp_seconds.